system/Common.php

```diff
@@ -75,7 +75,7 @@ function cache(?string $key = null)
         $cache = service('cache');
 
         // No params - return cache object
-        if ($key === null) {
+        if ((string) $key === '') {
```

Comment:
Seems like a change of behavior. What is wrong with checking `null`?

Reply:
Cache key cannot be empty
(https://github.com/codeigniter4/CodeIgniter4/blob/6cbbf601b047f1c60de3f9f7d4003b2aa46f3fa3/system/Cache/Handlers/BaseHandler.php#L60-L67)

Reply:
Exactly. An empty string should throw an exception and not return the cache instance.

Reply:
Good. I will cancel the update